metrics/ci_integrations/test/destination/firestore/config/validation_delegate/firestore_destination_validation_delegate_test.dart

```diff
@@ -68,16 +106,80 @@ void main() {
       return when(firebaseAuth.signIn(withEmail, withPassword));
     }
 
+    PostExpectation<Firestore> whenCreateFirestore() {
+      whenSignIn().thenAnswer((_) => Future.value(user));
+
+      return when(firestoreFactory.create(projectId, firebaseAuth));
+    }
+
+    PostExpectation<CollectionReference> whenReferenceCollection({
```

Comment:
```suggestion
    PostExpectation<CollectionReference> whenCollection({
```
Would it be a bit more readable?